gui/slick/js/editShow.js

```diff
@@ -1,80 +1,80 @@
 var allExceptions = [];
 
 function metaToBool(pyVar){
-    var meta = $("meta[data-var='" + pyVar + "']").data("content");
-    if(typeof meta === "undefined"){
-        console.log(pyVar + " is empty, did you forget to add this to main.mako?");
+    var meta = $('meta[data-var="' + pyVar + '"]').data('content');
+    if(typeof meta === 'undefined'){
+        console.log(pyVar + ' is empty, did you forget to add this to main.mako?');
         return meta;
     } else {
         meta = (isNaN(meta) ? meta.toLowerCase() : meta.toString());
-        return !(meta === "false" || meta === "none" || meta === "0");
+        return !(meta === 'false' || meta === 'none' || meta === '0');
     }
 }
 
 function getMeta(pyVar){
-    return $("meta[data-var='" + pyVar + "']").data("content");
+    return $('meta[data-var="' + pyVar + '"]').data('content');
 }
 
-var srRoot = getMeta("srRoot");
+var srRoot = getMeta('srRoot');
 
 
 $(document).ready(function(){
-    if (metaToBool("FANART_BACKGROUND")) {
-        $.backstretch(srRoot + "/showPoster/?show=" + $("#show").attr("value") + "&which=fanart");
-        $(".backstretch").css("opacity", getMeta("FANART_BACKGROUND_OPACITY")).fadeIn("500");
+    if (metaToBool('FANART_BACKGROUND')) {
+        $.backstretch(srRoot + '/showPoster/?show=' + $('#show').attr('value') + '&which=fanart');
+        $('.backstretch').css('opacity', getMeta('FANART_BACKGROUND_OPACITY')).fadeIn('500');
```

Comment:
Gotta draw a line somewhere. I don't want this to become a "lets drop all js fixes in here, PR"